README.md

```diff
@@ -384,7 +403,41 @@ server.listen(8080)
 
 ## Migrating to 2.x from 1.x
 
-The main change is the update to `path-to-regexp@2.0.0`, which has a few breaking changes:
+#### Deprecation of arity based error middleware
+
+In previous versions of `router` and `express`, the arity of a function was used to determine
+if a middleware was a normal middleware or an error handling middleware.  In `router@2.0` and `express@5`
+this behavior still works, but is deprecated.  The new api is as follows:
```

Comment:
So I'm a little confused by this deprecated statement. Do you mean that it still does the arity detection in .use .all .get etc. ?

Reply:
Yep! It just logs with `depd`.

Reply:
The idea being to remove it in `3.0` and consequently `express@6`.